packages/test-utils/jest-spyOn.ts

```diff
@@ -1,4 +1,4 @@
-export const jestMock = (object: any, method: string, implementation: () => any, accessType?: 'get' | 'set') => {
+export const jestSpyOn = (object: any, method: string, implementation: () => any, accessType?: 'get' | 'set') => {
```

Comment:
I don't think this wrapper is necessary since it's  just a shorthand for a chain calls.

Reply:
You can use chain calls directly, I think it is too long, so extract it.